lib/lucidboard/twiddler/actions.ex

```diff
@@ -17,10 +17,31 @@ defmodule Lucidboard.Twiddler.Actions do
          %Changeset{valid?: true} = cs <- Board.changeset(board, args),
          new_board <- Changeset.apply_changes(cs) do
       {:ok, new_board, fn -> Repo.update(cs) end, %{changeset: cs},
-       event("has updated the board settings.")}
+       event(board_event_message(args))}
     end
   end
 
+  defp board_event_message(%{"title" => title}) do
+    "has updated the board title to <b>#{title}</b>."
+  end
+
+  defp board_event_message(%{
+         "settings" => %{
+           "likes_per_user" => lpu,
+           "likes_per_user_per_card" => lpc
+         }
+       }) do
+    """
```

Comment:
This opens up a conversation about how we want to format, add emphasis, etc on event messages. It may not be super wise to assume we're outputting html...